src/main/java/net/dv8tion/jda/api/entities/MessageHistory.java

```diff
@@ -447,6 +447,36 @@ public static MessageRetrieveAction getHistoryAround(MessageChannel channel, Str
         return new MessageRetrieveAction(route, channel);
     }
 
+    /**
+     * Constructs a {@link net.dv8tion.jda.api.entities.MessageHistory MessageHistory} with the initially retrieved history
+     * of messages sent.
+     *
+     * <p>Alternatively you can use {@link net.dv8tion.jda.api.entities.MessageChannel#getHistoryFromBeginning(int) MessageChannel.getHistoryFromBeginning(...)}
+     *
+     * <h2>Example</h2>
+     * <br>{@code MessageHistory history = MessageHistory.getHistoryFromBeginning(channel).limit(60).complete()}
+     * <br>Will return a MessageHistory instance with the first 60 messages of the given {@link net.dv8tion.jda.api.entities.MessageChannel MessageChannel}.
+
+     *
+     * @param  channel
+     *         The {@link net.dv8tion.jda.api.entities.MessageChannel MessageChannel}
+     *
+     * @throws java.lang.IllegalArgumentException
+     *         If any of the provided arguments is {@code null};
+     *         Or if the provided messageId contains whitespace
```

Comment:
There is no messageId here

Reply:
Even with the `@Nonnull` Annotation, `null` should still be passable, shouldn't it? So only the line with the Message Id needs to be removed (and the first one needs to be adjusted)? I copied a similar JavaDoc for this function, thought I got rid of every wrong thing.. Thanks for noticing it and sorry.

Reply:
Yes you can still pass null, java doesn't have a typesystem that would allow denying this. The exception is thrown by `Checks.nonNull` when null is passed. Since there is no messageId parameter this part is unnecessary.